frontend/src/components/useClusterPoll.tsx

```diff
@@ -0,0 +1,20 @@
+import {useEffect, useState} from 'react'
+import {DescribeCluster} from '../model'
+
+export const useClusterPoll = (clusterName: string, startOnRender: boolean) => {
+  const [start, setStart] = useState(startOnRender)
+
+  useEffect(() => {
+    if (!start) return
```

Comment:
issue: should we clear the interval when `start` goes to false after `stop()` has been called?

Reply:
isn't this already taken care of of the code at line 13 ?

Reply:
Yes, it should be handled already by L13, otherwise I haven't understood your point